modules/custom/openy_upgrade_tool/src/ConfigUpdater.php

```diff
@@ -132,36 +125,49 @@ public function importConfigs(array $configs) {
     $this->filter($tmp_storage);
     // Import changed, just overwritten items, into config storage.
     $this->import($tmp_storage);
+    $_openy_config_import_event = FALSE;
   }
 
   /**
    * Simplified version of importConfigs.
    *
    * Main difference between this functions that in simple version we
    * skip export of all site configs to temp directory and just copy and import
-   * only listed config for import. Also here was skiped configs filter logic.
+   * only listed config for import. Also here was skipped configs filter logic.
    *
    * @param string $config
    *   Config name.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
    */
   public function importConfigSimple($config) {
     // Stream wrappers are not available during installation.
     $tmp_dir = (defined('MAINTENANCE_MODE') ? '/tmp' : 'temporary:/') . '/confi_simple' . $this->uuid->generate();
+    // Notify ConfigEventSubscriber that this is update from OpenY.
+    global $_openy_config_import_event;
```

Comment:
In previous version we used `openy_upgrade` option that was injected to config file (See `ConfigUpdater.php`). But I found new issue in latest drupal versions, on config save this option removed because not exist in config definitions. I have try to alter config object definition and provide addition value similar to `_core`, but this not work. So global statement is the fastest and easiest solution that work for now.

I agree that it needs to be redone, but it seems that I do not have time to do it to the next release date. Let's create separated issue and fix this later.